src/coreclr/src/System.Private.CoreLib/src/System/Reflection/Emit/MethodBuilder.cs

```diff
@@ -435,7 +435,7 @@ public override string ToString()
         #region MemberInfo Overrides
         public override string Name => m_strName;
 
-        internal int MetadataTokenInternal => GetToken().Token;
+        public override int MetadataToken => GetToken().Token;
```

Comment:
It would be nice to delete the unused `struct MethodToken` and `GetToken` methods.

Reply:
The implementation of GetToken can then move into this method.

Same for other types of the tokens.

Reply:
Once you do that, you can also delete it from `libraries\System.Reflection.Emit\src\MatchingRefApiCompatBaseline.txt`

Reply:
I pushed a commit that removes the structs. On one side, there is a bit of internal encapsulation lost, but on the other hand it removes an unnecessary internal abstraction that was only followed somewhat since the underlying Int32 was eventually used.

Some existing public methods (but not in the contract) that returned "Token" types now return Int32. I believe most could be made internal instead of public, but that was not done in the commit.

Reply:
>  I believe most could be made internal instead of public, but that was not done in the commit.

Yes, they should be internal.